server/test/integration/history.test.js

```diff
@@ -0,0 +1,56 @@
+import { createConnectedClient } from '../../src/db/dbClient.js';
+import sinon from 'sinon';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../server.js';
+import sinonChai from "sinon-chai";
+import { createTestUser, deleteTestUser }  from '../utils/userManagement.js';
+
+const { expect, assert } = chai;
+chai.use(sinonChai);
+chai.use(chaiHttp);
+
+describe('/api/history integration tests', () => {
+    const sandbox = sinon.createSandbox();
+    let testUserObj;
+    let pgClient;
+  
+    before(async () => {
+      pgClient = await createConnectedClient();
+    })
+  
+    beforeEach(async () => {
+      testUserObj = await createTestUser(pgClient, app);
+    });
+  
+    afterEach(async () => {
+      sandbox.restore();
+      await deleteTestUser(pgClient, testUserObj.userId);
+    });
+  
```

Comment:
in `beforeEach` and `afterEach`, you need to do buy & sell requests and cleanup in the db so that the user has some trading history data to be tested here. 

I don't see any addition of trading history data so these tests are working with **no** trading history. 

You can just use the `chai.request` methods here.